src/components/embedded/EmbeddedEventHeader.tsx

```diff
@@ -31,6 +31,7 @@ export default function EmbeddedEventHeader(props: Props) {
 	const { eventId, eventName, eventType, startTimestamp, endTimestamp } = event;
 
 	const status = getEventStatus(event);
+	const startTimestampValue = timestampToNumber(startTimestamp);
```

Comment:
lets change formatTime so it accepts string instead of importing timestampToNumber everywhere